core/src/bmi.jl

```diff
@@ -80,6 +80,7 @@ function BMI.initialize(T::Type{Model}, config::Config)::Model
     else
         storages, errors =
             get_storages_from_levels(parameters.basin, state.level)
+        storages = real.(storages)
```

Comment:
Is this to avoid dual numbers in your initial state?

Reply:
Yes, but it works without this